helm-chart/values.schema.json

```diff
@@ -476,24 +520,6 @@
       "required": ["samplerRatio", "exporters"]
     }
   },
-  "allOf": [
-  {
-    "if": {
-      "properties": {
-        "disableAutoInjection": {
-          "const": false
-        }
-      }
-    },
-    "then":{
-      "properties": {
-        "enabledNamespaces": {
-          "$ref": "#/definitions/emptyArray"
-        }
-      }
-    }
-  }
-],
   "anyOf": [
```

Comment:
This entire `anyOf` block can actually be removed. It was only valid when we had both tracing and telemetry.